SOURCES/xha-10.1.0-supports-drbd.XCP-ng.patch

```diff
@@ -0,0 +1,189 @@
+From 0c1da77c5e6a4bb8a7dd759b3ccd1beafe24b4cc Mon Sep 17 00:00:00 2001
+From: Ronan Abhamon <ronan.abhamon@vates.fr>
+Date: Thu, 6 Aug 2020 16:53:57 +0200
+Subject: [PATCH] feat(statefile): supports DRBD (close state file descriptor
```

Comment:
The commit message needs a description of the problem and of the solution implemented, as you would explain to me with enough details for me to understand.